### PR TITLE
[MERGE 10/14] HoC Banners and OG Tags

### PR DESCRIPTION
**Pull request by molly-moen**
"Revert Revert" of [37173](https://github.com/code-dot-org/code-dot-org/pull/37173) plus code from [37207](https://github.com/code-dot-org/code-dot-org/pull/37207) plus a minor update to `hourofcode.com/learn`.

Round 2 of Banner updates due to some build/timing issues. These should be merged on 10/14. Very minor changes from the original PR--most were outlined in [37207](https://github.com/code-dot-org/code-dot-org/pull/37207) which has been pulled into this PR. It also includes a tablet-sized image for `hourfocode.com/learn` since the desktop and mobile ones were not sufficient for tablets.

Final banner screenshots
`code.org` logged out homepage
Desktop
Screen Shot 2020-10-13 at 10 52 58 AM

Mobile
Screen Shot 2020-10-13 at 10 53 36 AM

`hourofcode.com` homepage
Desktop
Screen Shot 2020-10-13 at 10 54 36 AM

Mobile
Screen Shot 2020-10-13 at 10 54 06 AM

`hourofcode.com\learn`
Desktop
Screen Shot 2020-10-13 at 10 38 58 AM

Tablet
Screen Shot 2020-10-13 at 10 38 45 AM

Mobile
Screen Shot 2020-10-13 at 10 40 52 AM

See 37173 for banner screenshot--that is unchanged.

## Links

- [jira](https://codedotorg.atlassian.net/browse/PLC-997)
- [OG tag change spec](https://docs.google.com/document/d/1IxA8w-57JU8KwpsYsrXCnESCcXkbukPAPl1PDlaJEYA/edit#heading=h.utzlax1ku85m)
- [Banner spec](https://docs.google.com/document/d/1o59JSChSEeF-pvcQHrjmuLPL__5k0ruNWRrlpasNcbI/edit#)

## Testing story
Tested locally at multiple resolutions.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -33,6 +33,13 @@
   africa_campaign_date_year: "2020"
   africa_campaign_date_full_year: "October 1-31, 2020"
 
+  hoc2020_header: 'Learn today, build a brighter tomorrow.'
+  hoc2020_header_with_line_break_markdown: "Learn today,\n\nbuild a brighter tomorrow."
+  social_hoc2020_cs_important: 'CS is more important than ever. Let''s build the future we want. #CSforGood'
```
Comment (clareconstantine) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:38]: Two apostrophes in "Let''s" - should it be escaped?

**Review by tess323 — APPROVED** [review]
Looks great to me :)

**Review by clareconstantine — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (molly-moen) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:38]: the two apostrophes is to escape the apostrophe in Let's :)

Comment (clareconstantine) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:38]: haha ohhhh! thanks! I had no idea that's how escaping worked here :)

**Pull request merged**